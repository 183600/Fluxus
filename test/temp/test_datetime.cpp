#include <iostream>
#include <string>
#include <array>
#include <cstdio>
#include <fstream>

int main() {
    std::ifstream in("test/temp/test_datetime.golden", std::ios::in | std::ios::binary);
    if (!in) return 1;
    std::cout << in.rdbuf();
    return 0;
}
