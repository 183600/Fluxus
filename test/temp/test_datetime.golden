Current datetime: 2025-10-12 01:27:31.397717
Current date: 2025-10-12
Current time: 01:27:31.397717
Formatted datetime: 2025-10-12 01:27:31
Parsed datetime: 2023-12-25 15:30:00

Date arithmetic:
Today: 2025-10-12
Tomorrow: 2025-10-13
Yesterday: 2025-10-11
Next week: 2025-10-19

Days between 2023-01-01 and 2023-12-31: 364
Specific datetime: 2023-12-25 15:30:00
Unix timestamp: 1760203651.4084969

Sleeping for 0.5 seconds...
Done sleeping!

Date components:
Year: 2025
Month: 10
Day: 12
Hour: 1
Minute: 27
Second: 31
Today is Sunday

Time operations:
Noon time: 12:00:00
Time hour: 12
Time minute: 0